src/ssl.c

```diff
@@ -53643,4 +53643,23 @@ int wolfSSL_set_ephemeral_key(WOLFSSL* ssl, int keyAlgo,
 
 #endif /* WOLFSSL_STATIC_EPHEMERAL */
 
+#if defined(OPENSSL_EXTRA)
+void wolfSSL_THREADID_current(WOLFSSL_CRYPTO_THREADID* id)
```

Comment:
All wolfSSL API should have at least a one line comment describing what the function does, and in this case, why wolfSSL doesn't need any specific behavior here.

Reply:
Added comment